internal/ansible/controller/reconcile_test.go

```diff
@@ -409,34 +409,6 @@ func TestReconcile(t *testing.T) {
 					Namespace: "default",
 				},
 			},
-			ExpectedObject: &unstructured.Unstructured{
```

Comment:
This change is because of https://github.com/kubernetes-sigs/controller-runtime/pull/1399. When the finalizer is successfully run and is [removed](https://github.com/operator-framework/operator-sdk/blob/ad645e4bc6d6ae85fa5f0bffcad3bc871690608c/internal/ansible/controller/reconcile.go#L250), the fake client updates the object and immediately sends it for deletion.